contracts/truefi/LoanToken.sol

```diff
@@ -376,6 +376,14 @@ contract LoanToken is ILoanToken, ERC20 {
         return _balance().add(redeemed);
     }
 
+    function isRepaidInFull() external override view returns (bool) {
+        return _isRepaidInFull();
+    }
+
+    function _isRepaidInFull() internal view onlyOngoing returns (bool) {
```

Comment:
short, simple functions, we don't need an internal function. you can remove this and move the logic to the public one

Reply:
The other one is currently external (like all other functions in the ILoanToken interface), but I'll combine them and make it public.

What are your thoughts on the internal _balance() vs external balance() that just delegates? These are defined immediately below isRepaid().

Reply:
I just tried combining the internal/external isRepaid() functions into a single public function and got this compile error in ILoanToken:

    TypeError: Functions in interfaces must be declared external.

If I compile with a single external definition then (as expected) I get this error:

    DeclarationError: Undeclared identifier. Did you mean "Repaid" or "isRepaid"?
            if (isRepaid()) {

For now I'll keep the internal/external split, unless you know another way to resolve this?

Reply:
Ivan helped me figure it out. I hadn't realized the interface could have an external function that you can override with a public implementation.